app/scripts/modules/netflix/availability/availability.less

```diff
@@ -149,6 +152,9 @@
       padding: 15px 20px;
       color: #fff;
     }
+    .glyphicon {
```

Comment:
Outside the scope of this PR. The glyph icon comes from help-field, so should be a part of converting help-field.